### ISSUE TITLE
Which version of the plugin should I use?

### ISSUE THREAD

**Issue by volodya-lombrozo** [l3r8yJ/oop-cop#106]
In the README you have the following description of your plugin (how I can integrate it):
```xml
<build>
  <plugins>
    <plugin>
      <groupId>ru.l3r8y</groupId>
      <artifactId>oop-cop</artifactId>
      <executions>
        <execution>
          <goals>
            <goal>search</goal>
          </goals>
        </execution>
      </executions>
    </plugin>
  </plugins>
</build>
```

But here is the small problem - I don't know which version to use. It would be nice to have a version specified directly in the example:
```xml
<build>
  <plugins>
    <plugin>
      <groupId>ru.l3r8y</groupId>
      <artifactId>oop-cop</artifactId>
      <version>0.1.6</version>
      <executions>
        <execution>
          <goals>
            <goal>search</goal>
          </goals>
        </execution>
      </executions>
    </plugin>
  </plugins>
</build>
```

**Comment by zoeself** [comment]
@volodya-lombrozo thank you for reporting this. I'll assign someone to take care of it soon.

**Comment by zoeself** [comment]
@l3r8yJ this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2024-01-09T14:10:02.277126``.

Estimation here is ``60 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it [partially](https://docs.self-xdsd.com/microtasking.html) and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

If you don't have time or simply don't want to work on this, you can always [resign](https://docs.self-xdsd.com/projectmanager.html#commands).

**Comment by zoeself** [comment]
@l3r8yJ thank you for resolving this ticket. I've just added it to your active invoice. You can always check all your invoices and more on the [Contributor Dashboard](https://self-xdsd.com).
